rfcs/0005-rfc-process.md

```diff
@@ -0,0 +1,54 @@
+---
+created-on: 2021-03-29
+freshness: 2021-03-29
+owner: @dj2
+status: draft
+---
+
+[La version française suit.](#résumé)
+
+# RFC Process
+
+## Summary
+This RFC lays out some foundations for the RFC process, mostly around
+the approval and acceptance process.
+
+## Motivation
+Without some sort of framework it is difficult to know how the RFC
+process is going to work and to align everyone in the same direction.
+There are questions around when approvals are needed, how many, how long
+do we wait. This RFC attempts to set some guard rails for this process
+and to start us in the right direction.
+
+## Proposal
+1. A proposal, from PR announcement to commit MUST be no less then 2 weeks.
+   The announcement MUST be in Slack on the appropriate channels for the
+   RFC content.
+2. An RFC MUST have approval to commit. An RFC needs approval from the
+   communities which will be affected by the RFC (or, a best guess as to
+   the impacted communities).
+   (i.e. an RFC to setup base labels needs Dev, SRE and PM approval as
+   all three communities will need to agree)
```

Comment:
How many members of a community need to approve an RFC before it's ready to move to the next step.

Reply:
I like your suggestion above that 2 reviewers are required. I think a community just needs a single approver, but hopefully they've talked to their community about the RFC and aren't just deciding for themselves.